### PR TITLE
Bump kubevirtci k8s

### PR DESCRIPTION
**Pull request by qinqon**
The last kubevirtci does not work well with the functests at ovs-cni.
This change copy the ssh to node mechanism from kubernetes-nmstate that
is not to work fine, it also bump k8s to 1.23.

### CODE REVIEW

**Comment by phoracek** [comment]
@qinqon there seems to be a minor linting issue. Could you fix it?

File: tests/node/node.go [hunk 1]
```diff
@@ -21,44 +21,55 @@ package node
...
-	out, err = exec.Command("docker", "exec", string(out[:12]), "ssh.sh", command).CombinedOutput()
-	if err != nil {
-		panic(fmt.Sprintf("failed to run docker exec command error output: %s", string(out)))
-	}
-	outString := string(out)
-	outLines := strings.Split(outString, "\n")
-	// first two lines of output indicate that connection was successful
-	outStripped := outLines[2:]
-	outStrippedString := strings.Join(outStripped, "\n")
+func RunQuietAtNode(node string, command ...string) (string, error) {
```
Comment (phoracek) [inline tests/node/node.go:46]: Is this used anywhere?

File: tests/node/node.go [hunk 2]
```diff
@@ -21,44 +21,55 @@ package node
...
+	"github.com/k8snetworkplumbingwg/ovs-cni/tests/cmd"
 )
 
-// RunOnNode performs bash shell command on node
 // TODO: Use job with a node affinity instead
-func RunOnNode(node string, command string) (string, error) {
-	out, err := exec.Command("bash", "-c", fmt.Sprintf("docker ps | grep %s | awk '{ print $1}'", node)).CombinedOutput()
-	if err != nil {
-		panic(fmt.Sprintf("failed to run docker ps error output: %s", string(out)))
+func runAtNodeWithExtras(node string, quiet bool, command ...string) (string, error) {
```
Comment (phoracek) [inline tests/node/node.go:33]: Once you drop unused RunQuietAtNode, the quiet flag is no longer needed

**Review by phoracek — CHANGES_REQUESTED** [review]

File: tests/node/node.go (hunk 1 above)
Comment (qinqon) [reply tests/node/node.go]: Removing

File: tests/node/node.go (hunk 2 above)
Comment (qinqon) [reply tests/node/node.go]: ditto

**Pull request merged**